src/config_app.erl

```diff
@@ -48,5 +49,18 @@ command_line() ->
 
 default() ->
     Etc = filename:join(code:root_dir(), "etc"),
-    Default = [filename:join(Etc,"default.ini"), filename:join(Etc,"local.ini")],
+    Default = [
+        filename:join(Etc, "default.ini"),
+        filename:join(Etc, "default.d"),
+        filename:join(Etc, "local.ini"),
+        filename:join(Etc, "local.d")
+    ],
     lists:filter(fun filelib:is_file/1, Default).
+
+expand_dirs(File) ->
+    case filelib:is_dir(File) of
+        true ->
+            filelib:wildcard(File ++ "/*.ini");
```

Comment:
the current code does a glob though, and no sort, so that would be a change, albeit a good one. still want the short?